File: Selenium/Activity3.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {

	public static void main(String[] args) {

		WebDriver driver = new FirefoxDriver();

		driver.get("https://training-support.net/webelements/login-form");
		String title = driver.getTitle();
		System.out.println("Title of the page is: " + title);
		driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
		driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
		driver.findElement(By.xpath("	")).click();

		String message = driver.findElement(By.xpath("//h1[contains(text(),'Success!')]")).getText();
		System.out.println("Message after login is: " + message);
		
		driver.quit();

		
//		  driver.get("https://training-support.net/");
//		  
//		  String title1=driver.getTitle();
//		  System.out.println("Title of the page is: "+title1);
//		
//		  driver.findElement(By.xpath("//a[text()='About Us']")).click(); String
//		  newTitle1=driver.getTitle();
//		  System.out.println("New Title of the page is: "+newTitle1);
//		  
//		  driver.quit();
		 

	}

}

File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form")
    title=driver.title
    print("Title of the page is: ",title)

    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")
    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")
    driver.find_element(By.XPATH, "//button[text()='Submit']").click()

    message=driver.find_element(By.XPATH,"//h1[contains(text(),'Success!')]").text
    print("Message after login is: ",message)

    # driver.get("https://training-support.net/")

    # title=driver.title
    # print("Title of the page is: ",title)

    # driver.find_element(By.XPATH, "//a[text()='About Us']").click()
    # newTitle=driver.title
    # print("New Title of the page is: "+newTitle)
